src/lib/hooks/useCurrency.ts

```diff
@@ -121,7 +122,11 @@ export function useTokenFromActiveNetwork(tokenAddress: string | undefined): Tok
     () => decimals.loading || symbol.loading || tokenName.loading,
     [decimals.loading, symbol.loading, tokenName.loading]
   )
-  const parsedDecimals = useMemo(() => decimals.result?.[0], [decimals.result])
+  const parsedDecimals = useMemo(
+    () => (decimals.result ? decimals.result?.[0] : DEFAULT_ERC20_DECIMALS),
```

Comment:
Should this be something like decimals?.result?.[0] ?? DEFAULT_ERC20_DECIMALS, or are you trying to explicitly preserve a `null` value?

Reply:
yeah that makes sense actually, done.